### DISCUSSION TITLE
Inlet profile overridden by assigned boundary condition

### DISCUSSION THREAD

**Discussion by eskimmel** [su2code/SU2#1649] [Q&A]
Hi all,

I'm caught in a catch 22 regarding specified inlet profiles. I have created a specified inlet profile that I would like to use, and I followed the instructions [here](https://su2code.github.io/tutorials/Inc_Laminar_Step/) to create it. I used the automatically generated `example_inlet.dat` file and pasted in my own profile values, so I am confident that it is formatted correctly. I have `SPECIFIED_INLET_PROFILE= YES` and `INLET_FILENAME= example_inlet.dat` in my configuration file, and the output even reads `Reading inlet profile from file: example_inlet.dat`. However, I am still forced to assign values to `MARKER_SUPERSONIC_INLET` (Mach 6 flow) or else the simulation aborts. When I enter values for `MARKER_SUPERSONIC_INLET`, then these values are used for my inlet conditions rather than my specified inlet profile. The beginning of my domain is horizontal flow over a no-slip wall, so SU2 overriding my specified inlet profile with a uniform profile is causing my simulation to diverge at the inlet. Is there a way to get around this?

**Comment by pcarruscag** [comment]
Hi, unfortunately the supersonic inlet does not support inlet profiles at the moment.
You can create a feature request, and if you are willing to write a bit of code we can point to what to change.

**Reply by eskimmel** [reply]
Hi, thanks for the quick response. I will create a feature request for this. I'd be happy to help implement this change, though I have very little experience with developing/C++. If you would please point out which files should be adjusted, I can start working on it.
